resources/test_data/tbl/meta_tables/meta_segments_accurate_libcpp.tbl

```diff
@@ -5,5 +5,5 @@ int_int|0|1|b|int|2|null|null|200|2|6|0|0|0
 int_int|1|0|a|int|1|null|null|200|1|3|0|0|0
 int_int|1|1|b|int|1|null|null|200|1|3|0|0|0
 int_int_int_null|0|0|a|int|2|RunLength|null|144|0|12|0|0|0
-int_int_int_null|0|1|b|int|1|Dictionary|BitPacking|108|0|4|0|0|4
```

Comment:
why is the test data adapted? I think it would create better documentation to comment out tests that are currently failing with an explanation of why they are out-commented

Reply:
e.g. shouldn't this work without adapting test data?

```cpp
INSTANTIATE_TEST_SUITE_P(VectorCompressionTypes, StorageDictionarySegmentTest,
                         ::testing::Values(VectorCompressionType::FixedWidthInteger /*,VectorCompressionType::BitPacking, currently unsupported*/ ),
                         dictionary_segment_test_formatter);
```